### PR TITLE
Updated Preprocessing module

### PR DESCRIPTION
**Pull request by Bradley-Karat**
Added in a new blip-up blip-down figure into the preprocessing module as well as a few additional sentences

### CODE REVIEW

File: _episodes/preprocessing.md [hunk 1]
```diff
@@ -17,17 +17,19 @@ start: true
...
-This dataset contains single shell diffusion data with 7 $b = 0 s/mm^2$ volumes
-(non-diffusion weighted) and 60 $b = 1000 s/mm^2$ volumes. In addition, field
-maps (found in the <code>fmap</code> directory are acquired with opposite
+Diffusion MRI data does not typically come off the scanner ready to be analyzed,
+and there can be many things that might need to be corrected before analysis. 
+Diffusion preprocessing typically comprises of a series of steps to perform the 
+necessary corrections to the data. These steps may vary depending on how the data 
+is acquired. Some consensus has been reached for certain preprocessing steps, 
+while others are still up for debate. The lesson will primarily focus on the 
+preprocessing steps where consensus has been reached. Preprocessing is performed
```
Comment (kaitj) [inline _episodes/preprocessing.md:26]: Should note that it is the preprocessing in this notebook as there are other available tools/packages.

File: _episodes/preprocessing.md [hunk 2]
```diff
@@ -17,17 +17,19 @@ start: true
...
+necessary corrections to the data. These steps may vary depending on how the data 
+is acquired. Some consensus has been reached for certain preprocessing steps, 
+while others are still up for debate. The lesson will primarily focus on the 
+preprocessing steps where consensus has been reached. Preprocessing is performed 
+using a few well-known software packages (e.g. [FSL](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki), [ANTs](https://github.com/ANTsX/ANTs). For the purposes 
+of these lessons, preprocessing steps requiring these software packages has already 
+been performed for the dataset <code>ds000221</code> and the commands required for 
+each step will be provided. This dataset contains single shell diffusion data with 
+7 b = 0 s/mm^2 volumes (non-diffusion weighted) and 60 b = 1000 s/mm^2 volumes. 
+In addition, field maps (found in the <code>fmap</code> directory are acquired with opposite
```
Comment (kaitj) [inline _episodes/preprocessing.md:32]: I don't remember if field maps have been introduced before either in diffusion or intro to MRI, but would be good to have a brief description if not

**Review by kaitj — COMMENTED** [review]
Couple of minor comments, and similar to the the introductory module, should run this through a linter.

**Review by jhlegarreta — COMMENTED** [review]
Thanks for doing this Brad, and sorry for the late review.

When addressing Jason's comments, please squash all commits into a single commit, or else, do separate commits for typo and grammar correction vs new content (blip-up/down).

**Comment by jhlegarreta** [comment]
@Bradley-Karat would you have time to address the comments or do you prefer to hand it over to somebody else? Thanks.

**Comment by jhlegarreta** [comment]
@Bradley-Karat would you have time to address the comments or do you prefer to hand it over to somebody else? Thanks.

File: _episodes/preprocessing.md [hunk 3]
```diff
@@ -171,7 +176,7 @@ are used. Briefly:
 regarding the acquisition.
 - <code>--config=b02b0.cnf</code> makes use of a predefined config file.
 supplied with <code>topup</code>, which contains parameters useful to
-registering with good $b = 0 s/mm^2$ images.
+registering with good b = 0 s/mm^2 images.
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply _episodes/preprocessing.md:179]: Contradicts my comment https://github.com/carpentries-incubator/SDC-BIDS-dMRI/pull/222#discussion_r1292365193. Keeping them should be preferred I think.

**Comment by jhlegarreta** [comment]
So I realize that PR #222 has incorporated the changes in this PR, right @Bradley-Karat ? In that case, this PR should be closed. Thanks for your work.

**Comment by jhlegarreta** [comment]
@Bradley-Karat can you please address the comments made?

**Pull request closed without merging**